src/Giraffe/EndpointRouting.fs

```diff
@@ -13,7 +13,7 @@ open Giraffe
 
 module private RouteTemplateBuilder =
     let private guidPattern =
-        "([0-9A-Fa-f]{{8}}\-[0-9A-Fa-f]{{4}}\-[0-9A-Fa-f]{{4}}\-[0-9A-Fa-f]{{4}}\-[0-9A-Fa-f]{{12}}|[0-9A-Fa-f]{{32}}|[-_0-9A-Za-z]{{22}})"
+        "^[0-9A-Fa-f]{{8}}-[0-9A-Fa-f]{{4}}-[0-9A-Fa-f]{{4}}-[0-9A-Fa-f]{{4}}-[0-9A-Fa-f]{{12}}$|^[0-9A-Fa-f]{{32}}$|^[-_0-9A-Za-z]{{22}}$"
```

Comment:
The last part `|^[-_0-9A-Za-z]{{22}}$` matches for a 22 char GUID e.g `0000000000000000000000` but there´s no test for it. Is there a format for such a GUID?

Reply:
It's this [ShortGuid](https://github.com/giraffe-fsharp/Giraffe/blob/master/src/Giraffe/ShortGuid.fs) format. I'll add a test for it.

Reply:
Done with this commit https://github.com/giraffe-fsharp/Giraffe/pull/594/commits/b57fac833b4cb78b9524f1c9a1ee267d765e6ac2